### DISCUSSION TITLE
Calling for Community Experts

### DISCUSSION THREAD

**Discussion by squidfunk** [squidfunk/mkdocs-material#6363] [Announcement]
We're launching a new program called [Community Experts](https://squidfunk.github.io/mkdocs-material/insiders/community-experts-program/) to reward awesome users that regularly contribute in discussions and help other users to solve their problems with free access to Insiders! If you want to join, get in touch with us!

__Update__: currently no open seats.

**Comment by kamilkrzyskow** [comment]
Interesting course of action, as after the introduction of the AI Chatbot in the search I noticed a decline in the amount of created discussions, and Alex Voss also is typically first to respond and I think he maintains and expands the example repository.

Therefore, I am surprised you need more helpers that each need to answer 5 discussions a month, as currently in the top user chart there are 18 answers listed in the last 30 days. 

I feel tempted to try, but at the same time I don't think I'll be able to keep up with Alex's pace of answering and other Community Experts to meet the quota.

**Reply by squidfunk** [reply]
Excellent observation. @alexvoss is part of our team now and is currently managing the discussion board in order to get to know the project better. However, in the coming months, the team needs to focus on other things, e.g., improving the documentation, building examples and pushing the entirety of the project forward. It's still mainly me that does the development, and every task that I can delegate allows me to put more time into interesting stuff, like the new search.

The project is growing steadily, and with the next funding goals being hit, and the major overhaul of search, it's likely that adoption will grow further. For this reason, we wanted to see whether there are individuals that can help us out on supporting the community. You are definitely one of the prime candidates, given how much you've helped other users already. Furthermore, [we've closed our Gitter channel](https://squidfunk.github.io/mkdocs-material/blog/2023/10/02/sunsetting-gitter-towards-efficient-community-engagement/) and concentrate our efforts on GitHub Discussions as the only support channel.

Note that the quota is not a hard limit. Essentially, we'd like to see ~15 contributions in 3 months. If there are weeks where you can't find the time, no problem. Ideally, just let us know (you'll be invited to a private team), and others can compensate. If, in the end, there were only 14 contributions in 3 months, it's also not a deal breaker. I hope you understand that we just need to set some expectation, and we thought that 5 per month is quite fair. If you don't think this is the case, I'd be eager to learn what you'd think would be fair.

Also note that there have been 18 answered discussions in the last 30 days, so we're currently aiming to onboard 3 community experts. Additionally, there are still several unanswered discussions.

**Reply by squidfunk** [reply]
Also, for reference, here're some statistics of our discussion board for the last 12 months:

<img width="809" alt="Bildschirm­foto 2023-11-19 um 16 49 24" src="https://github.com/squidfunk/mkdocs-material/assets/932156/a187b2a3-39f1-4cf1-b6ca-4714e5d33284">

**Reply by kamilkrzyskow** [reply]
Thanks for the statistics.
I see that there was an uplift in discussions in September so the Chatbot was implemented just in time to meet growing demand.
I now also understand that you want to focus on development and let Alex focus on creating examples or something else, and let Community Experts answer all of the discussions.

I understand that evaluating contribution isn't so simple and the answers were chosen as a good enough metric, because it's a part of GitHub itself.

However, like mentioned in documentation there will be harder and easier issues to solve, and Community Experts might gravitate to the easier ones to get easy contribution points. Not even in a malicious manner just more time efficient.

Another issue is no real indication that there is someone in the process of answering, no assignee like for GitHub Issues. Because of that there might be situations where 2-3 Experts are working on the same issue and answer at the same time. One will be chosen as the answer and the other one, I assume, will be noted by your team as a valid contribution, but I fear there still might be cases of "wasted effort".

---

Having said all that I read the documentation page once again, and noticed:
> Please note that the role of a Community Expert cannot be fulfilled by an organization member seeking access for the entire organization's benefit.

and unfortunately this means I'm not a good fit for this, as I don't really need Insiders features for small personal projects, but they would be beneficial for the Gothic Modding Community website, which is deployed from a GitHub organization, where currently 2 people would have access to the access token.

**Reply by squidfunk** [reply]
Fair enough, thanks for your view on this!

**Reply by squidfunk** [reply]
@kamilkrzyskow we've revised the organization requirement – you can build projects hosted in organizations. Access will be granted to your personal account, for which you can generate an access token and use that token inside GitHub Actions. That way, you can build your organization's documentation using Insiders, as long as it is non-commercial.

**Comment by iBug** [comment]
Any chance this gets extended to other aspects? Like issues and PRs? I'd love to see activity there being honored.

For PRs there's a simple enough criterion, but I'm not sure how contributions in the Issues board can be evaluated.

**Reply by squidfunk** [reply]
Thanks for suggesting. We're starting with the discussion board, but will consider to extend this to other aspects in the future. Issues and PRs need careful evaluation, since there's no binary classification as for discussions (= answered or not answered). PRs that introduce new code (don't fix bugs) bind team resources, which is what we're trying to free with the program.

If a user asks a questions in an issue (which is the wrong medium to begin with, but it happens quite often), and the issue can be resolved with the help of a community expert, it counts towards the quota. We'll convert such issues to discussions. Same goes for PRs that fix known or unknown bugs that are opened or co-authored by community experts.

However, please understand that we currently cannot extend this to features.

**Comment by sushma26-dev** [comment]
Hi,
I am using mkdocs material. Facing some issues after build. Its not gets redirected properly after mkdocs build. But its works fine locally.
Can anyone help me to solve the issue.

**Reply by alexvoss** [reply]
Please start a new discussion and post details about what does not work, ideally a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/). With enough information to go by one of us usually manages to help.

**Reply by sushma26-dev** [reply]
yes, can able to build it properly. But the issue is , after build  its created a folder called site. When i open the index.html its gets opened in browser. When I try to redirect to another page **ex: About page**. Its gets open **C://folderName/about** , but what i need it to gets open with **C://folderName/about/index.html**. I tried using nav in md file, but its won't work.

**Reply by alexvoss** [reply]
Have a look at the [offline plugin](https://squidfunk.github.io/mkdocs-material/plugins/offline/#built-in-offline-plugin). The documentation contains some notes about the limitations involved in loading a site from the local filesystem.

**Reply by sushma26-dev** [reply]
Thanks for your support. **use_directory_urls: false** help me to solve the issue.

**Comment by niclasheinz** [comment]
> Update: only 2 seats left.

Is that still up to date?

**Reply by squidfunk** [reply]
Thanks for your interest in the program. As mentioned [here](https://squidfunk.github.io/mkdocs-material/insiders/community-experts-program/#ready-to-get-started), could you please send us an email at community@squidfunk.com?
